packages/flutter/test/widgets/framework_test.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/material.dart';
```

Comment:
Consider sorting these imports?

Reply:
Removing that import. Auto-import of unimported libraries gets a bit confused in the flutter sdk.

Reply:
Checks for unused imports also appear to be confused likely by re-exported methods that are used. For example, both material and widget re-export some methods from foundation that are indeed used. Fixing that issue would be a bit subtle.